napari/utils/theme.py

```diff
@@ -91,13 +94,35 @@ def _ensure_syntax_style(value: str) -> str:
         )
         return value
 
+    @validator("font_size", pre=True)
+    def _ensure_font_size(value: str) -> str:
+        assert "pt" in value, trans._(
```

Comment:
Suggestion to be stricter on where pt is ? I assume if it's not at the end it will fail anyway so up to you.
```suggestion
        assert value.endswith('pt'), trans._(
```

Reply:
Makes sense to me :+1: